setup.py

```diff
@@ -73,7 +73,7 @@ def get_version(package):
 SCRIPTS = glob('bin/scripts/*')
 
 INSTALL_REQUIRES = ['argparse', 'bottle', 'paste', 'pyserial']
-INSTALL_REQUIRES += ['pyelftools']
+INSTALL_REQUIRES += ['pyelftools', 'intelhex', 'python-magic']
```

Comment:
Those new packages don't seem to be used in the changes. Are they really required ?

Reply:
it's due to the cc2538 dependency. 
python-magic is not fully needed, I think I'll just write temporary firmwares to flash with a .hex extension and not add python-magic.

Reply:
you mean the cc2538-bsl.py tool ? this is an external tool so no need to add this dependency to the gateway code. I think it's enough to just have it as a dependency in the docker container and a note in the readme (like it's already done)

Reply:
You're right